v2/fake/fake.go

```diff
@@ -224,3 +225,23 @@ type UnbindReaction struct {
 	Response *v2.UnbindResponse
 	Error    error
 }
+
+func strPtr(s string) *string {
+	return &s
+}
+
+func AsyncRequiredError() error {
```

Comment:
You should add tests for these methods to ensure they correctly trip the helper functions in errors.go